docs/_includes/default.liquid

```diff
@@ -25,38 +25,14 @@
 
     <main id="content">{{ content }}</main>
 
-    <aside class="sponsorship">
-      <a
-        href="https://matomo.org/"
-        rel="external noopener"
-        title="Mocha is sponsored by Matomo"
-      >
-        <img src="images/matomo-logo.png?trim" loading="lazy" alt="Matomo logo" />
+    <div class="sponsorship">
+      <a href="https://matomo.org/" rel="external noopener" title="Mocha is sponsored by Matomo">
+        <img src="images/matomo-logo.png?trim" loading="lazy" width="100" alt="Matomo logo">
       </a>
-      <a
-        title="Mocha is an OpenJS Foundation Project"
-        href="https://openjsf.org"
-        rel="external noopener"
-      >
-        <img
-          src="/images/openjsf-logo.svg"
-          loading="lazy"
-          width="300"
-          height="94"
-          alt="OpenJS Foundation Logo"
-        />
+      <a href="https://wallabyjs.com" rel="external noopener" title="Mocha is sponsored by Wallaby" id="wallaby-logo">
+        <img src="https://wallabyjs.com/assets/img/logoWithText.svg" loading="lazy" width="140" alt="Wallaby logo">
```

Comment:
Some logos are loaded out of the `images` folder, some by external links.
What's the difference? Performance?

Reply:
I think there is no reason. 
Whether we manage all assets on our side or not is only difference. Performance won't make much difference on mocha site.